src/object/camera.hpp

```diff
@@ -106,6 +106,8 @@ class Camera final : public GameObject,
 
   /** smoothly slide the scale of the camera towards a new value */
   void ease_scale(float scale, float time, easing ease);
+
+  Path* get_path_if_exists() override { return get_path(); }
```

Comment:
Same (Also, you placed this function inside the functions for scripting, between the Doxygen braces)